src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
  for (let action of actions) {
```
Можна було б використати цикл for of. Індекси нам тут не потрібні, а виглядає коротше і користуватись зручніше. Хоча і так ок